### DISCUSSION TITLE
NodeSeek 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#159] [模板讨论]
NodeSeek 模板问题反馈评论区
有问题可以下面反馈。

**Comment by 6396036** [comment]
脚本失效了，求更新

**Reply by wjf0214** [reply]
更新了，改为cookie模式了。

**Reply by lazureice** [reply]
遇到CF的 "Sorry, you have been blocked". 
用了cookie版也一样。

**Reply by wjf0214** [reply]
提示你被屏蔽了，是不是你账号或者IP有问题？我没遇到过。

**Comment by ghost** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: OpenSSL SSL_connect: Connection reset by peer in connection to www.nodeseek.com:443 ,
Request URL: https://www.nodeseek.com/

**Reply by wjf0214** [reply]
检测检测网络连接情况，访问网站失败了。

**Comment by Grouppppp** [comment]
报错 http code 403，测试发现结果为 CF 的 "Sorry, you have been blocked"，浏览器手动访问是正常的。大佬可否修一修？

2024.07.12 update: 修改模板，http header 增加 User-Agent 后恢复。

**Comment by miaoxiaozhao** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.nodeseek.com/

**Comment by yanaxiao** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.nodeseek.com/

提示这个是什么错？

**Comment by miaoxiaozhao** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Recv failure: Connection reset by peer,
Request URL: https://www.nodeseek.com/

连不上了，之前用代理还可以的，现在也不行了。

**Comment by 6396036** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 599: Connection timed out after 30000 milliseconds,
Request URL: https://www.nodeseek.com/

求更新

**Comment by ZDY321** [comment]
~~~yaml
Failed at 1/12 request,Fail assert: {"re": "200", "from": "status"} from success_asserts,Response Error : HTTP 403: Forbidden,Request URL: https://www.nodeseek.com/
--
~~~
大佬，求更新

**Comment by icaca** [comment]
大佬，求更新

**Comment by fu5502** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.nodeseek.com/


求大佬感谢 🙏

**Comment by foxdodo** [comment]
加了盾好像过不去了

**Comment by musug** [comment]
Failed at 1/12 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://www.nodeseek.com/
求大佬感谢 🙏

**Comment by hecarli555** [comment]
国内本地的话提示599，在境外vps上提示403，。。。怎么破？

**Comment by Willywang8216** [comment]
無了

**Comment by hecarli555** [comment]
这个签到还能修吗？
